### PR TITLE
#58 Support for multiple elementTypes per CreateOperationHandler

### PR DESCRIPTION
**Pull request by tortmayr**
Necessary adaptions foreclipse-glsp/glsp-server/pull/55

### CODE REVIEW

File: server/org.eclipse.glsp.example.workflow/src/main/java/org/eclipse/glsp/example/workflow/handler/CreateActivityNodeHandler.java [hunk 1]
```diff
@@ -27,12 +27,16 @@
...
    private final String label;
+   private final String elementTypeId;
 
    public CreateActivityNodeHandler(final String elementTypeId, final String label) {
       super(elementTypeId);
+      this.elementTypeId = elementTypeId;
       this.label = label;
    }
 
+   protected String getElementTypeId() { return elementTypeId; }
```
Comment (planger) [inline server/org.eclipse.glsp.example.workflow/src/main/java/org/eclipse/glsp/example/workflow/handler/CreateActivityNodeHandler.java:38]: Shouldn't we push this into `CreateNodeOperationHandler`?

Comment (planger) [reply server/org.eclipse.glsp.example.workflow/src/main/java/org/eclipse/glsp/example/workflow/handler/CreateActivityNodeHandler.java:38]: And, if you agree, also for `CreateEdgeOperationHandler` I guess...

Comment (tortmayr) [reply server/org.eclipse.glsp.example.workflow/src/main/java/org/eclipse/glsp/example/workflow/handler/CreateActivityNodeHandler.java:38]: I'm not sure if this is generally applicable. In the case of the example specific handlers we know that we always have one element type id and never change this via setter. This assumption does not hold for the generic handlers

Comment (tortmayr) [reply server/org.eclipse.glsp.example.workflow/src/main/java/org/eclipse/glsp/example/workflow/handler/CreateActivityNodeHandler.java:38]: But this catched another issue. I forgot to add the new constructor for multiple element type to `CreateNodeOpreationHandler` and `CreateEdgeOperationHandler`. I will create a follow up PR for this

**Pull request merged**